client/python/tests/integration/test_no_auth.py

```diff
@@ -105,6 +105,18 @@ def create_queue(client: ArmadaClient, queue_name):
     wait_for(client, queue=queue_name)
 
 
+def test_batch_update_and_create_queues(client: ArmadaClient):
+    # Need to separately create queue name so that it is not
+    # automatically created by the fixture.
+    queue_name = f"queue-{uuid.uuid1()}"
+
+    queue = client.create_queue_request(name=queue_name, priority_factor=1)
+    client.create_queues([queue])
```

Comment:
You should have a check to verify that the queues were actually created.  

I also think you should have a multiple list queue and check each entry to verify that the queue creation worked.